src/main/java/org/isf/menu/service/MenuIoOperations.java

```diff
@@ -175,45 +166,49 @@ public boolean isGroupNamePresent(String groupName) throws OHServiceException {
 
 	/**
 	 * Inserts a new {@link User} in the DB
-	 * 
 	 * @param user - the {@link User} to insert
 	 * @return the new {@link User} added to the DB
 	 * @throws OHServiceException When failed to create user
 	 */
 	public User newUser(User user) throws OHServiceException {
+		user.setDeleted(false);
 		return repository.save(user);
 	}
 
 	/**
 	 * Updates an existing {@link User} in the DB
-	 * 
 	 * @param user - the {@link User} to update
 	 * @return new {@link User}
 	 * @throws OHServiceException When failed to update user
 	 */
 	public boolean updateUser(User user) throws OHServiceException {
+		ensureUserNotDeleted(user.getUserName());
 		return repository.updateUser(user.getDesc(), user.getUserGroupName(), user.getUserName()) > 0;
```

Comment:
Please see https://openhospital.atlassian.net/browse/OP-1130

`new` operations should return the object (which these files seem to do)
`update` operations should return the object (not a boolean as found in these files)
`delete` operations should return null (which these files seem to do)

It appears that I missed updating all the JPA operations in this package when I did all the others.  Sorry